### PR TITLE
Team 2

### PR DESCRIPTION
**Pull request by bjohnst1998**
Team-2's Friday pull request

### CODE REVIEW

**Comment by sophiagavrila** [comment]
Conflicting with Team-3's StagingListener.java and implementation classes

**Review by sophiagavrila — COMMENTED** [review]
I believe StaginListenerImpl was restructured and places in `util` directory, rather than `AssignmentManagerUtil` directory. @RevatureGentry can you check that this looks alright?

File: src/main/java/com/revature/backend/util/StagingListener.java [hunk 1]
```diff
@@ -1,15 +1,18 @@
-package com.revature.backend.AssignmentManagerUtil;
+package com.revature.backend.util;
```
Comment (sophiagavrila) [inline src/main/java/com/revature/backend/util/StagingListener.java:1]: @RevatureGentry `AssignmentManagerUtil` was renamed to `util`, but Git is preventing a merge.  I'm unsure how to resolve this

**Comment by bjohnst1998** [comment]
That is correct @sophiagavrila The folder structure was reorganized. Which is why Team-2's pull should be the last one merged in.

**Review by sophiagavrila — COMMENTED** [review]
understood, thank you @bjohnst1998 - my mistake in jumping ahead

**Review by sophiagavrila — DISMISSED** [review]

**Review by sophiagavrila — APPROVED** [review]
resolved all renaming conflicts in testing and `src/ main/java/backend/util`

**Pull request merged**